src/libraries/Microsoft.PowerFx.Core/Public/Types/AggregateType.cs

```diff
@@ -122,11 +122,18 @@ public IEnumerable<NamedFormulaType> GetFieldTypes()
         {
             return FieldNames.Select(field =>
             {
-                var fieldType = GetFieldType(field);
-                
                 var displayName = DType.TryGetDisplayNameForColumn(_type, field, out var dName)
                     ? dName
                     : null;
+
+                if (this.TryGetBackingDType(field, out var backingDType))
+                {
+                    var t = new TypedName(backingDType, new DName(field));
+                    return new NamedFormulaType(t, displayName);
+                }
+
+                var fieldType = GetFieldType(field);
```

Comment:
>fieldType [](http://example.com/codeflow?start=20&length=9)

This is not used, but this should not matter, if above was false, this would throw... 
Most likely we can remove it with if/else I mentioned